pkg/state/scripts_storage.go

```diff
@@ -344,15 +344,15 @@ func (ss *scriptsStorage) newestAccountHasVerifier(addr proto.WavesAddress, filt
 		return script.HasVerifier(), nil
 	}
 	script, err := ss.newestScriptAstByKey(keyBytes, filter)
-	if err != nil {
+	if err != nil { // TODO: check error type
```

Comment:
Don't remove this TODOs, actually there's possible two types of errors NotFound-like and state failures. First one must be swallowed and replaced by `false, nil`, the second have to be passed upward.

Reply:
my bad